src/main/java/org/synyx/urlaubsverwaltung/calendar/CalendarProperties.java

```diff
@@ -20,11 +23,37 @@
     @Email
     private String organizer;
 
+    /**
+     * Defines a refresh interval for iCal Feed.
+     * This property specifies a suggested minimum interval for polling for changes of the calendar data from
+     * the original source of that data.
+     * <p>
+     * Possible inputs:
+     * <p><ul>
+     * <li>30 (30 Minutes)
+     * <li>PT30M (30 Minutes in ISO-8601 Format)
+     * <li>30m (30 Minutes)
+     * </ul><p>
+     * Default refresh interval is one day (P1D).
```

Comment:
sollte hier PT1D stehen?

Reply:
Ich glaube erst ab Stunden/Minuten/Sekunden Angaben:
https://www.digi.com/resources/documentation/digidocs/90001437-13/reference/r_iso_8601_duration_format.htm

Reply:
Ist es dann in der Reame falsch?!